### DISCUSSION TITLE
摆脱能开发出mac版本吗?刚从win换到mac,想用mouseInc,结果没法打开exe文件

### DISCUSSION THREAD

**Discussion by QingjiaTsang** [shuax/MouseInc.Settings#252] [General]
摆脱能开发出mac版本吗?刚从win换到mac,想用mouseInc,结果没法打开exe文件.

**Comment by shuax** [comment]
我没有mac，也不会mac开发，用别的类似软件吧，我见过。

**Reply by iamansen** [reply]
我出费用恳请作者开发mac版本！！硬件没有我提供！！win里面太好用了，mac类似的软件用过了不好用，恰好我工作需要经常用mac 没有作者的软件我工作效率下降很多 ，再次恳请作者，看能否满足！谢谢致敬！

**Reply by shuax** [reply]
确实满足不了，我都不会mac

**Reply by Bush2021** [reply]
> 我出费用恳请作者开发mac版本！！硬件没有我提供！！win里面太好用了，mac类似的软件用过了不好用，恰好我工作需要经常用mac 没有作者的软件我工作效率下降很多 ，再次恳请作者，看能否满足！谢谢致敬！

试试 BetterAndBetter? 勉强可以算是无缝切换

**Reply by iamansen** [reply]
Popclip 、BetterAndBetter 都用过了 操作繁琐 反应迟钝 很不好用，寻求Mac开发者 我付费开发 有没有推荐的呀
